parm/config/gfs/config.resources

```diff
@@ -635,6 +647,18 @@ case ${step} in
         ntasks_quilt=${ntasks_quilt_gfs}
         nthreads_fv3=${nthreads_fv3_gfs}
         nthreads_ufs=${nthreads_ufs_gfs}
+        # Will not be set if we are skipping the mediator
+        nthreads_mediator=${nthreads_mediator_gfs:-}
+      elif [[ "${_CDUMP}" =~ "gdas" ]]; then
+        export layout_x=${layout_x_gdas}
+        export layout_y=${layout_y_gdas}
+        export WRITE_GROUP=${WRITE_GROUP_GDAS}
+        export WRTTASK_PER_GROUP_PER_THREAD=${WRTTASK_PER_GROUP_PER_THREAD_GDAS}
+        ntasks_fv3=${ntasks_fv3_gdas}
+        ntasks_quilt=${ntasks_quilt_gdas}
+        nthreads_fv3=${nthreads_fv3_gdas}
+        nthreads_ufs=${nthreads_ufs_gdas}
+        nthreads_mediator=${nthreads_mediator_gdas:-}
```

Comment:
Yeah, they need to stay `_gfs` and `_gdas` for now since `config.ufs` only defines variables with `_gfs` and `_gdas` for both deterministic (`fcst`) and ensemble (`efcs`) jobs.  Thus, there isn't a, e.g., `layout_x_enkfgdas` variable to assign to `layout_x`.

Reply:
Thanks!